templates/jobs/main.html

```diff
@@ -40,8 +38,12 @@
                         Jobs
                     </div>
                     <div class="panel-body">
-                        <p>Here you can find job offers for an entry-level
-                            positions.</p>
+                        <p>
+                            Working as a software developer is a great way to become one. There is lots of companies out there who are hiring, but how to tell which one is good?
+                        </p>
+                        <p>
+                            We talked to a lot of them and here you can find offers from companies who hire beginners and want to train them. They’re not targeting women only, but they do care about diversity in a workplace. We trust they do the right things.
```

Comment:
We have talked to a lot of them and here you can find offers from companies who hire beginners and want to train them. They’re not targeting women only, but they do care about diversity in a workplace. We trust they do the right things.